ldes-server-admin/ldes-server-admin-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/kafkasource/KafkaSourceRepository.java

```diff
@@ -0,0 +1,11 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.admin.domain.kafkasource;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.KafkaSourceProperties;
+
+import java.util.List;
+
+public interface KafkaSourceRepository {
+	void save(KafkaSourceProperties kafkaSource);
+	void delete(String id);
```

Comment:
As discussed, the delete method could be deleted after an FK is added to this entity